UNITTESTS/stubs/mbed_rtc_time_stub.cpp

```diff
@@ -0,0 +1,48 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2013 ARM Limited
```

Comment:
Can you add SPDX identifier here and also fix the year

Reply:
sure